web/themes/custom/drevops/assets/sass/brand/_tokens.scss

```
@@ -0,0 +1,306 @@
+// (from the design's [data-color-scheme="dark"] block) directly at
+// :root. The Google Fonts @import is intentionally omitted (fonts are
+// already loaded by the theme).
+//
+
+// ══════════════════════════════════════════════════════════════
+//    PALETTE
+//    Eleven derived steps per brand colour. Step 1 = lightest,
+//    step 11 = near-black. The brand base colour sits near step 9.
+//    ══════════════════════════════════════════════════════════════
+:root {
+  /* primary */
```

Stylelint (17.12.0)

[error] Expected // for comments instead of /* (scss/comment-no-loud)

(scss/comment-no-loud)